features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -1793,6 +1793,7 @@
         {% if authentication.jit_provisioning.allow_idp_login is defined %}
             <AllowLoginToIDP>{{authentication.jit_provisioning.allow_idp_login}}</AllowLoginToIDP>
         {% endif %}
+        <AllowAssociatingToExistingUser>{{authentication.jit_provisioning.associating_to_existing_user}}</AllowAssociatingToExistingUser>
```

Comment:
Shall we define the default value for this config in default.json file?

Reply:
```suggestion
        <AllowAssociatingToExistingUser>{{authentication.jit_provisioning.associating_to_existing_user | default(false)}}</AllowAssociatingToExistingUser>
```

Since this is just a boolean value defaulting to false, without adding this to the default.json file shall we add inline default value as above?

Reply:
I feel it is better if we add this to default json. Sometimes, I prefer to check default.json file to look for configurations as most of the default values of the configs are available there.

Reply:
added to default.json file through https://github.com/wso2/carbon-identity-framework/pull/5468/commits/fb117f2cd78ccb6cad0eb90a069840bbe90ffdcb